### PR TITLE
[framework] translate constraint messages in yaml files in config directory

### PR DESCRIPTION
**Pull request by sspooky13**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Functionality to translate custom constraint messages for GraphQL. It's possible Frontend and Backend validation are not exactly same, so you wanna FE developers to be informatied about what this is issue and they can show error message customer. P.S.: czech language is not my native speak language so please check czech translate. Thanks
|New feature| Yes
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No
|Fixes issues| ...
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by vitek-rostislav** [comment]
Hi Peter, thanks for the pull request, I have rebased your branch and added upgrade notes (removing the translation message ids is considered to be a BC break, see https://docs.shopsys.com/en/9.1/contributing/backward-compatibility-promise/#translation-messages). Besides that, I do not have any remarks, so let's wait for the second review from one of my colleagues :wink:

**Review by vitek-rostislav — APPROVED** [review]

**Review by grossmannmartin — APPROVED** [review]
really cool! reviewed and tested. Thank you 👍

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**
